db/migrate/20200521150213_create_storage_services.rb

```diff
@@ -0,0 +1,18 @@
+class CreateStorageServices < ActiveRecord::Migration[5.1]
+  def change
+    create_table :storage_services do |t|
```

Comment:
This sounds like it might be closer to a storage profile?  We also have a storage_profiles model (it is pretty sparse right now but could be expanded out if it does match)

Reply:
@agrare , @Fryguy, what do you think? StorageService is also something that's supposed to provision a volume on demand. Would it make sense for there to be `StorageProfile.create_cloud_volume()` method? Or is it better if we have a separate `StorageService` that provides `StorageService.create_cloud_volume()`?

Reply:
If it is something that's going to be shown in the service catalog I'm okay with not reusing StorageProfile.  But in that case should this be an actual Miq Service (aka what we show in our service catalog)?  cc @Fryguy

Reply:
I think this would be a backend for an actual MIQ service. 
The idea is that we would sync these StorageServices from the provider during inventory scan. 
When a user wants a new volume, the miq-service-catalog would have a dropdown where they select the StorageService.
A request would then be send to the provider, asking it to create a new pool on this service. 

Also, since it is collected as part of an inventory, it needs ems_id.